### PR TITLE
Configuring rustfmt to only reorder imports in group

### PR DESCRIPTION
**Pull request by Rhuagh**

### CODE REVIEW

File: amethyst_config/rustfmt.toml [hunk 1]
```diff
@@ -1 +1,4 @@
 max_width = 200
+reorder_imports = true
+reorder_import_names = true
+reorder_imports_in_group = true
```
Comment (Xaeroxe) [inline amethyst_config/rustfmt.toml]: Can you also stick these in /rustfmt.toml?  There's no need for amethyst_config to have formatting different from the rest of the repository right?

Comment (Rhuagh) [reply amethyst_config/rustfmt.toml]: They are already, in atleast one of the other PRs.

Comment (Rhuagh) [reply amethyst_config/rustfmt.toml]: I can remove the config specific config file.

Comment (Xaeroxe) [reply amethyst_config/rustfmt.toml]: That would be preferable

**Comment by Rhuagh** [comment]
Ok, config file is moved.

File: rustfmt.toml [hunk 2]
```diff
@@ -1 +1,3 @@
 reorder_imports = true
+reorder_import_names = true
```
Comment (Xaeroxe) [inline rustfmt.toml]: This was already done by a previous PR, can you remove this commit from the branch?

**Comment by Rhuagh** [comment]
Removed :)

**Pull request merged**